TRAINING_IN_DETAIL.rst

```diff
@@ -0,0 +1,285 @@
+.. _header-n795:
+
+Using SageMaker Containers for SageMaker Training
+=================================================
+
+This document discusses in detail how to use SageMaker Containers for training. 
+
+.. contents::
+
+.. _header-n860:
+
+
+Building and training your own algorithm container using SageMaker Containers
+-----------------------------------------------------------------------------
+
+SageMaker Containers makes easier the process to Build your own container (BYOC). In this scenario, ``SAGEMAKER_PROGRAM``, containing the name of the
```

Comment:
s/Build/build

also "BYO" usually stands for "bring your own" 😂

Reply:
Fixed.